packages/wdio-sauce-service/src/service.ts

```diff
@@ -86,6 +84,12 @@ export default class SauceService implements WebdriverIO.ServiceInstance {
             this._suiteTitle = test.fullName.slice(0, test.fullName.indexOf(test.description) - 1)
         }
 
+        /* istanbul ignore if */
+        if (this._browser && !this._isUP && !this._isJobNameSet) {
+            this._browser.execute('sauce:job-name=' + this._suiteTitle)
```

Comment:
We can't do this here because the suite title never changes between tests. You probably want to put this in the before hook?

Reply:
AFAIK the before hook doesn't have the title for Jasmine, that's why I've put it there. It also is only going to executed once

Reply:
ah ok, we should remove `/* istanbul ignore if */` and have a test for this

Reply:
Ahh, chips

It needed to be ignore else. There are 2 tests already added

Reply:
Changed it to else